themis/getting-started.md

```diff
@@ -7,15 +7,14 @@ title:  Getting started
 
 ## Availability
 
-Themis supports multiple CPU architectures:
-x86_64/i386, ARM, various Android architectures.
+Themis supports big-endian CPU architectures: x86_64/i386, ARM, various Android architectures.
```

Comment:
1. They are little-endian, in fact.
2. I don't think there are reasons to emphasize this.